### ISSUE TITLE
latents full of 'nan'

### ISSUE THREAD

**Issue by tyxbarry** [jx-wang-s-group/CoNFiLD#7]
Hello!
When using my own data(10000,10886,1) and coords(10886,2), a hollow square velocity field, the cnf training step got the results full of nan. 
What caused it and how should I solve it?
Thanks.

**Comment by Xin-yang-Liu** [comment]
Hi, 

Thanks for your interest in our work. However, we cannot debug on your specific data. But here are some suggestions for you to start debugging.

1. Please always check if your data contain any abnormal value (e.g. `NaN`, `inf`)
2. Check if you installed the same version of packages (e.g. Numpy, PyTorch) 
3. Try a smaller learning rate

Thanks

**Comment by tyxbarry** [comment]
邮件已收到，谢谢您的邮件，我会尽快查收回复。
